openverse_catalog/dags/providers/provider_dag_factory.py

```diff
@@ -169,6 +169,7 @@ def create_provider_api_workflow(
     execution_timeout: timedelta = timedelta(hours=12),
     doc_md: Optional[str] = "",
     media_types: Sequence[str] = ("image",),
+    create_preingestion_tasks: Optional[Callable] = None,
```

Comment:
Do you mind adding documentation to the docstring about this new parameter?

Reply:
Oops, re-upping this! It'd be great to have documentation on these two new params in the docstring.

Reply:
Oops! Sorry, done.